### PR TITLE
First time login info

### LINKED ISSUE #9: First time login
Present the user with his infobox

### PR DESCRIPTION
**Pull request by emilps**
Edit info pops up first time you login

closes #9

### CODE REVIEW

File: src/components/EditUserDialog.jsx [hunk 1]
```diff
@@ -76,7 +77,7 @@ class EditUserDialog extends Component {
     return (
       <div>
         <Dialog
-          title="User profile"
+          title="Are your information correct?"
```
Comment (oleob) [inline src/components/EditUserDialog.jsx]: Is your information correct?*

**Review by oleob — CHANGES_REQUESTED** [review]

File: src/components/EditUserDialog.jsx (hunk 1 above)
Comment (emilps) [reply src/components/EditUserDialog.jsx]: fixed

**Review by emilps — COMMENTED** [review]
correct now?

**Review by oleob — APPROVED** [review]
verri good

**Pull request merged**